podcasts/CustomTimeStepper.swift

```diff
@@ -36,7 +36,11 @@ class CustomTimeStepper: UIControl {
     var bigIncrements: TimeInterval = 5.minutes
     var smallIncrements: TimeInterval = 1.minute
     var smallIncrementThreshold: TimeInterval = 5.minutes
-    var currentValue: TimeInterval = 1.hour
+    var currentValue: TimeInterval = 1.hour {
+        didSet {
+            accessibilityValue = L10n.timeShorthand(Int(currentValue))
```

Comment:
We can use the `localizedTimeDescription` to read out the full time value instead of the shorthand. 

```suggestion
            accessibilityValue = currentValue.localizedTimeDescription ?? ""
```

Reply:
Here's a video of the values VO is reading out:

https://user-images.githubusercontent.com/793774/203174415-6792c994-2e5d-4d49-a504-f0d0a4bc7012.mov

Reply:
Hey @emilylaguna good that you caught it. I will get on it probably in a few days. This is something that I really want to get right.

Reply:
Hey @emilylaguna I finally got around to fixing this. Sorry for the huge delay. Please check it out.